doc/man1/flux-job.rst

```diff
@@ -103,24 +103,35 @@ Wait for job(s) to complete and exit with the largest exit code.
 WAIT
 ====
 
-A waitable job may be waited on with ``flux job wait``.  A specific job
-can be waited on by specifying a jobid.  If no jobid is specified, the
-command will wait for any waitable user job to complete, outputting that
-jobid before exiting.  This command will exit with error if the job is not
-successful.
-
-Compared to ``flux job status``, there are several advantages /
-disadvantages of using ``flux job wait``.  For a large number of jobs,
-``flux job wait`` is far more efficient, especially when used with the
-``--all`` option below.  In addition, job ids do not have to be specified
-to ``flux job wait``.
-
-The two major limitations are that jobs must be submitted with the
-waitable flag, which can only be done in user instances.  In addition,
-``flux job wait`` can only be called once per job.
+``flux job wait`` behaves like the UNIX :linux:man2:`wait` system call,
+for jobs submitted with the ``waitable`` flag.  Compared to other methods
+of synchronizing on job completion and obtaining results, it is very
+lightweight.
+
+The result of a waitable job may only be consumed once.  This is a design
+feature that makes it possible to call ``flux job wait`` in a loop until all
+results are consumed.
+
+.. note::
+  Only the instance owner is permitted to submit jobs with the ``waitable``
+  flag.
+
+When run with a jobid argument, ``flux job wait`` blocks until the specified
+job completes.  If the job was successful, it silently exits with a code of
+zero.  If the has failed, an error is printed on stderr, and it exits with
```

Comment:
If the has failed -> If it has failed?  If the job has failed?

Reply:
Thanks - fixed tthat one but realized I should also note that `flux job wait` currently fails for both a failed job and no more waitable jobs.